Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -0,0 +1,93 @@
+{
+    public class ResponsesMessageService
+    {
+        private const string DESCRIPTION = "dokumentacja:\nhttps://watchman.readthedocs.io/pl/latest/135-services-in-framework/";
+        private readonly ResponsesGetterService _responsesDatabase;
+        private readonly EmbedMessageSplittingService _embedMessageSplittingService;
+
+        public ResponsesMessageService(ResponsesGetterService responsesDatabase, EmbedMessageSplittingService embedMessageSplittingService)
+        {
+            _responsesDatabase = responsesDatabase;
+            _embedMessageSplittingService = embedMessageSplittingService;
+        }
+
+        public async Task PrintResponses(string commandArgument, ulong serverId)
+        {
+            if (commandArgument == "default")
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Domyślne responses:", DESCRIPTION, GetDefaultResponses());
+            }
+            else if (commandArgument == "custom")
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Nadpisane responses:", DESCRIPTION, GetCustomResponses(serverId));
+            }
+            else
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Wszystkie responses:", DESCRIPTION, GetAllResponses());
+            }
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetDefaultResponses()
+        {
+            var responses = _responsesDatabase.GetResponsesFromBase()
+                            .Where(x => x.IsDefault)
+                            .ToList();
+
+            foreach (var response in responses)
+            {
+                yield return new KeyValuePair<string, string>(response.OnEvent, GetMessageWithoutFramesAndBold(response.Message));
+            }
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetCustomResponses(ulong serverId)
+        {
+            var responses = _responsesDatabase.GetResponsesFromBase()
+                            .Where(x => !x.IsDefault)
+                            .Where(x => x.ServerId == serverId)
+                            .ToList();
+
+            if (responses.Any() == false)
+            {
+                yield return new KeyValuePair<string, string>("-------------------------------", "```Obecnie jest brak nadpisanych responses!```");
+            }
+            foreach (var response in responses)
+            {
+                yield return new KeyValuePair<string, string>(response.OnEvent, GetMessageWithoutFramesAndBold(response.Message));
+            }
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetAllResponses()
+        {
+            var responses = _responsesDatabase.GetResponsesFromBase().ToList();
+
+            foreach (var response in responses)
+            {
+                yield return new KeyValuePair<string, string>(response.OnEvent, GetMessageWithoutFramesAndBold(response.Message));
```

Comment:
znowu to samo

ToList a później mapowanie z yield
nie ma to żadnego sensu

Reply:
czyli zrobić po prostu samo `return responses`?

Reply:
return responses.Select(x => new KeyValuePair<string, string>(response.OnEvent, GetMessageWithoutFramesAndBold(response.Message));